### PR TITLE
Store card

### LINKED ISSUE #17: shop list page

### PR DESCRIPTION
**Pull request by glu8637**
Shop page 1.0 complete
Doesn't have functionality yet

### CODE REVIEW

**Comment by nonorze-VOXX** [comment]
npm run format

File: frontend/src/routes/+page.svelte [hunk 1]
```diff
@@ -1,8 +1,217 @@
...
+					<Card.Description>Card Description</Card.Description>
+				</Card.Header>
+				<Card.Content>
+					<p>Card Content</p>
+				</Card.Content>
+				<Card.Footer>
+					<p>Card Footer</p>
+				</Card.Footer>
+			</Card.Root>
+		</div>
```
Comment (nonorze-VOXX) [inline frontend/src/routes/+page.svelte]: let it be a component or use {#each}

File: frontend/src/routes/+page.svelte [hunk 2]
```diff
@@ -1,8 +1,217 @@
...
+			</div>
+		</div>
+
+		<p style="padding-left: 10%; font-size: large;">Price</p>
+		<div>
+			<p>
+				NT$1 &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;NT$700
+				&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;NT$1000
+			</p>
+			<Slider value={[50]} max={100} step={1} class="max-w-[85%]" />
```
Comment (nonorze-VOXX) [inline frontend/src/routes/+page.svelte]: <div class="flex justify-between">

File: frontend/src/routes/+page.svelte [hunk 3]
```diff
@@ -1,8 +1,217 @@
...
+			<div>
+				<div>
+					<Checkbox id="fresh" />
+					<Label for="fresh">Fresh</Label>
+					<Checkbox id="organic" />
+					<Label for="organic">Organic</Label>
+					<Checkbox id="healthy" />
+					<Label for="healthy">Healthy</Label>
+					<Checkbox id="dairy-free" />
+					<Label for="dairy-free">Dairy-Free</Label>
```
Comment (nonorze-VOXX) [inline frontend/src/routes/+page.svelte]: use {#each}

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]

File: frontend/src/routes/+page.svelte [hunk 4]
```diff
@@ -1,8 +1,110 @@
...
+					<Label for={id}>{id}</Label>
+				</div>
+			{/each}
+		</div>
+
+		<p style="padding-left: 10%; font-size: large;">Price</p>
+		<div style="display:flex; padding-right:20%">
+			<div style="flex:1">NT$1</div>
+			<div style="flex:1">NT$700</div>
+			<div style="flex:1">NT$1000</div>
```
Comment (nonorze-VOXX) [inline frontend/src/routes/+page.svelte]: flex:1 look like is no need

File: frontend/src/routes/+page.svelte [hunk 5]
```diff
@@ -1,8 +1,110 @@
...
-<p>Visit <a href="./food" class="text-blue-400"> Shop list</a></p>
+<div style="width: 40%; overflow: hidden;">
+	<div class="text-2xl" style="float: left;">PUN Street Access</div>
+</div>
+
+<div style="display: flex; justify-content: flex-start;">
+	<div style="width: 40%; float: left">
+		<Input type="text" placeholder="QQㄋㄟㄋㄟ好喝到咩噗茶" class="max-w-xs" />
+
+		<p style="padding-left: 10%; font-size: large;">Sort</p>
```
Comment (nonorze-VOXX) [inline frontend/src/routes/+page.svelte]: padding-left => class pl-1
font-size => text-lg

File: frontend/src/routes/+page.svelte [hunk 6]
```diff
@@ -1,8 +1,110 @@
...
+			<div style="max-width: 60%;">
+				{#each tagCheckboxes as { id }}
+					<Checkbox {id} />
+					<Label for={id}>{id}</Label>
+				{/each}
+			</div>
+		</div>
+	</div>
+
+	<div style="display: flex-wrap; max-width: 70%">
```
Comment (nonorze-VOXX) [inline frontend/src/routes/+page.svelte]: display: flex-wrap => class flex-wrap

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]
remove style with class of tailwind. chat GPT will help you with that
move those shop card to routes/shops/+page.svelte
you will see change in localhost:5173/shops

File: frontend/src/routes/+page.svelte (hunk 4 above)
Comment (glu8637) [reply frontend/src/routes/+page.svelte]: need flex to generate whitespace and even spacing

Comment (glu8637) [reply frontend/src/routes/+page.svelte]: update: indeed dont need after changing to tailwind class

File: frontend/src/routes/+page.svelte (hunk 5 above)
Comment (glu8637) [reply frontend/src/routes/+page.svelte]: fix

File: frontend/src/routes/+page.svelte (hunk 6 above)
Comment (glu8637) [reply frontend/src/routes/+page.svelte]: fix

Comment (nonorze-VOXX) [reply frontend/src/routes/+page.svelte]: i think you have not push

Comment (glu8637) [reply frontend/src/routes/+page.svelte]: pushed

**Review by nonorze-VOXX — APPROVED** [review]
LG

**Pull request merged**